board_types.txt

```diff
@@ -62,6 +62,7 @@ AP_HW_MRO_CONTROL_ZERO_OEM            142
 AP_HW_MATEKF765_WING                  143
 AP_HW_JDMINIF405                      144
 AP_HW_KAKUTEF7_MINI                   145
+AP_HW_NIGHTCRAWLER                    146
```

Comment:
According to the comment on line 33, this should really be at a high number, above 1000.

Is it going to be a real pain from your end to change things?